lib/providers/device_provider.dart

```diff
@@ -80,6 +80,22 @@ abstract class DeviceProvider extends ChangeNotifier {
     }
   }
 
+  sdk.State<dynamic>?
```

Comment:
I like all these helper methods. What do you guys think about eventually moving them inside of the `Trait` class found in the SDK?

Reply:
I think this is probably a good idea

Reply:
Would we want to do this now or as a part of a refactor? I have a couple things I want to change that could all go in to one pr.